marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.mjs

```diff
@@ -0,0 +1,129 @@
+/**
+ Copyright 2012-2019 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+import * as LoadData from './loadData';
+
+const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
```

Comment:
Shouldn't we retain support for 9.x servers as well ? If so, wouldn't there be issues running mjs in 9.x servers ?

Reply:
This is for 5.3, which I believe we said that we'll be requiring 10.0-3